rotkehlchen/accounting/structures/types.py

```diff
@@ -67,6 +67,7 @@ class HistoryEventType(SerializableEnumNameMixin):
     INFORMATIONAL = auto()
     MIGRATE = auto()
     RENEW = auto()
+    MARGIN = auto()
```

Comment:
Can you remove this then since it's not used? Let's maybe add it when we need it. Not sure yet how we would like to represent it. On the type or subtype.